src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -1608,6 +1615,71 @@ abstract class ManagedStrategy(
         }
     }
 
+    private fun methodAnalysisSectionType(
+        owner: Any?,
+        className: String,
+        methodName: String,
+        atomicMethodDescriptor: AtomicMethodDescriptor?,
+        deterministicMethodDescriptor: DeterministicMethodDescriptor<*, *>?,
```

Comment:
What is `DeterministicMethodDescriptor`?

Reply:
It is related to native method calls support by `@zhelenskiy`